CONTRIBUTING.md

```diff
@@ -145,3 +145,16 @@ Before submitting a pull request, please make sure that:
 ## Contributing documentation
 
 Please format and lint documentation with `task docs`.
+
+## Contributing code on Windows
+
+In order to contribute code on Windows, please make sure that:
+
+* You have a Windows machine with `git` installed.
+* You have Go (1.18 and above) installed directly on your machine and have `go` in your PATH.
+* You have [WSL2](https://docs.microsoft.com/en-us/windows/wsl/) (Windows Subsystem for Linux) installed.
+* You have [Docker for Windows](https://docs.docker.com/desktop/windows/install/) installed.
+
+To run all commands you should use Windows command prompt.
+WSL2 distros are not supported yet.
+Setting up developer environment is pretty much the same as described [above](#Setting up the development environment).
```

Comment:
If we move the whole section up, there is no need to make it a link

Reply:
Removed the whole sentence.